components/Workspaces/WorkspaceBanner.tsx

```diff
@@ -9,9 +9,9 @@ type WorkspaceBannerProps = {
 export default function WorkspaceBanner({ workspaceId, openModal }: WorkspaceBannerProps) {
   const posthog = usePostHog();
 
-  useEffectOnce(() => {
+  useEffect(() => {
```

Comment:
This should be the same thing as `useEffectOnce`. `useEffectOnce` is an alias for this.